openedx/course-discovery
# feat: Support multiple verified seats for a single course run

## Problem Statement
Mobile team has started adding additional two verified seats(One for Android and second for iOS) on ecommerce for a single run of a course. When these changes are pushed into discovery through publish api they are producing [errors](https://splunk.edx.org/en-US/app/search/search?q=search%20index%3D%22prod-edx%22%20course_discovery.apps.course_metadata.models.Seat.MultipleObjectsReturned%3A%20get()%20returned%20more%20than%20one%20Seat%20--%20it%20returned%202!&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=1684627200&latest=1684886400&sid=1686037847.1378796) and creating problems while publishing the course.

## Solution
This PR is adjusting places where discovery is expecting only a single verified seat for a course run. 
We are planning to add third variant either through ecommerce api or using discovery django admin.

## Testing instructions

Go to http://localhost:18381/admin/course_metadata/seat/ and add a third seat for any course(make sure to add draft version too). Now with master branch if you try to change price of the course in publisher it will give you error and you will not be able to publish it. Move to this branch and it course will be published successfully.

[Learner-9425](https://2u-internal.atlassian.net/browse/LEARNER-9425)
